client/src/components/Hom.js

```diff
@@ -0,0 +1,45 @@
+
+import React from "react";
+// import logo from "/Users/nagehan/Documents/hugsforbugs/client/src/components/logo.png";
+import vid from "./pexels-c-technical-6334253.mp4"
+import "./heading.css" ;
+import { Link } from "react-router-dom";
+import Button from 'react-bootstrap/Button';
+import Headings from "./Headings";
+import Subheading from "./Subheading";
+
+function Hom() {
+  return (
+    <div>
+      {/* <img
+					className="logo"
+					data-qa="logo"
+					src={logo}
+					alt="Just the React logo"
+				/>
+ */}
+
```

Comment:
Just a couple of things here - We need to rename and format the file - it has a spelling mistake (should be Home.js)
you can right-click the code it in vscode and select formatting:
![image](https://user-images.githubusercontent.com/38047481/220451942-880c93a0-95ee-46d2-bf72-65b1d77cb1ff.png)